vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -910,6 +911,47 @@ public void fetchCurrentUser(ModelCallback<User> callback) {
         fetchContent(Vimeo.ENDPOINT_ME, CacheControl.FORCE_NETWORK, callback);
     }
 
+
+    /**
+     * A generic synchronous GET call. Unless implementing your own threading,
+     * use {@link #fetchContent(String, CacheControl, ModelCallback, String, Map, String)}
+     *
+     * @param uri           URI of the resource to GET
+     * @param cacheControl  Cache control type (includes max age and other cache policy information)
+     * @param query         Query string for hitting the search endpoint
+     * @param refinementMap Used to refine lists (generally for search) with sorts and filters
+     * @param fieldFilter   The string of fields to include in the response (highly recommended!)
+     * @return the Retrofit {@link retrofit2.Response} object
+     */
+    @Nullable
+    public retrofit2.Response<Object> fetchContentSync(String uri, CacheControl cacheControl,
+                                                       @Nullable String query,
+                                                       @Nullable Map<String, String> refinementMap,
+                                                       @Nullable String fieldFilter) {
+        if (uri == null || uri.isEmpty()) {
+            throw new AssertionError("uri cannot be null or empty");
+        }
+        String cacheHeaderValue = createCacheControlString(cacheControl);
+
+        Map<String, String> queryMap = new HashMap<>();
+        if (refinementMap != null && !refinementMap.isEmpty()) {
+            queryMap = refinementMap;
+        }
+        if (query != null && !query.isEmpty()) {
+            queryMap.put(Vimeo.PARAMETER_GET_QUERY, query);
+        }
+        if (fieldFilter != null && !fieldFilter.isEmpty()) {
+            queryMap.put(Vimeo.PARAMETER_GET_FIELD_FILTER, fieldFilter);
+        }
+
+        Call<Object> call = this.vimeoService.GET(getAuthHeader(), uri, queryMap, cacheHeaderValue);
+        try {
+            return call.execute();
+        } catch (IOException e) {
+            return null;
+        }
+    }
```

Comment:
I think we should pull the rest of this method out into a helper (in addition to the cache control jazz) just because these two methods will always be identical except for 1. having a callback and 2. how it gets executed (so the top and the bottom). Maybe with one for each object we're modifying or maybe just one that modifies both objects.